src/Query/Builder.php

```diff
@@ -1251,6 +1260,21 @@ protected function compileWhereRaw(array $where): mixed
         return $where['sql'];
     }
 
+    protected function invalidOperator($operator)
+    {
+        if (! is_string($operator)) {
+            return true;
+        }
+
+        $operator = strtolower($operator);
+
+        if (isset($this->conversion[$operator])) {
```

Comment:
I noticed this PR didn't introduce the `$conversion` property, it just added to it. How is this used in the base Eloquent library?

I'm also curious because you basically recreated the entire `$conversion` map in this PR (assuming `'=' => '='` is redundant and should be removed), so I'm unsure if it's still relevant to any parent context.

Reply:
`$conversion` is specific to mongodb and doesn't exist in Eloquent.

Yes `'=' => '='` is useless. I removed it and restored the complete operators list in `$operators` because the property is public. So I removed this method overload.